### ISSUE TITLE
Uncaught TypeError: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by Camypaper** [OmniSharp/omnisharp-atom#107]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.192.0
**System**: Unknown Windows Version
**Thrown From**: [omnisharp-atom](https://github.com/OmniSharp/omnisharp-atom) package, v0.2.1

### Stack Trace

Uncaught TypeError: Cannot read property 'split' of undefined

```
At C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\node_modules\text-buffer\lib\range.js:40

TypeError: Cannot read property 'split' of undefined
  at Function.module.exports.Range.fromText (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\node_modules\text-buffer\lib\range.js:40:19)
  at TextBuffer.module.exports.TextBuffer.buildPatch (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\node_modules\text-buffer\lib\text-buffer.js:477:24)
  at TextBuffer.module.exports.TextBuffer.setTextInRange (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\node_modules\text-buffer\lib\text-buffer.js:455:20)
  at TextBuffer.module.exports.TextBuffer.setText (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\node_modules\text-buffer\lib\text-buffer.js:387:19)
  at TextEditor.module.exports.TextEditor.setText (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\src\text-editor.js:682:26)
  at RenameView.module.exports.RenameView.configure (C:\Users\camypaper\.atom\packages\omnisharp-atom\lib\omnisharp-atom\views\rename-view.coffee:19:31)
  at Rename.module.exports.Rename.rename (C:\Users\camypaper\.atom\packages\omnisharp-atom\lib\omnisharp-atom\features\rename.coffee:22:19)
  at atom-workspace.<anonymous> (C:\Users\camypaper\.atom\packages\omnisharp-atom\lib\omnisharp-atom\features\rename.coffee:11:63)
  at atom-workspace.handler (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\src\space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\node_modules\space-pen\vendor\jquery.js:4681:9)
  at atom-workspace.elemData.handle (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\node_modules\space-pen\vendor\jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\src\command-registry.js:238:29)
  at C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\src\command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\node_modules\atom-keymap\lib\keymap-manager.js:519:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\node_modules\atom-keymap\lib\keymap-manager.js:354:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (C:\Users\camypaper\AppData\Local\atom\app-0.192.0\resources\app\src\window-event-handler.js:180:20)

```

### Commands

```
     -3:25.1 editor:newline (atom-text-editor.editor)
     -3:24.0 core:backspace (atom-text-editor.editor)
     -3:20.3 core:move-left (atom-text-editor.editor)
     -3:19.1 core:save (atom-text-editor.editor)
     -3:17.9 markdown-preview:toggle (atom-text-editor.editor)
     -3:13.8 core:backspace (atom-text-editor.editor)
     -3:13.6 core:delete (atom-text-editor.editor)
     -3:10.0 core:save (atom-text-editor.editor)
  3x -3:00.8 core:undo (atom-text-editor.editor)
     -2:53.3 core:save (atom-text-editor.editor)
  4x -2:50.7 core:move-left (atom-text-editor.editor)
     -2:49.1 core:save (atom-text-editor.editor)
     -2:44.6 settings-view:open (atom-text-editor.editor)
     -0:12.3 omnisharp-atom:rename (div.settings-view.pane-item)
     -0:08.3 core:cancel (div.settings-view.pane-item)
  2x -0:04.3 omnisharp-atom:rename (div.settings-view.pane-item)
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v2.10.0
autocomplete-snippets, v1.2.0
file-icons, v1.5.4
japanese-wrap, v0.2.7
omnisharp-atom, v0.2.1
Sublime-Style-Column-Selection, v1.2.3

# Dev
No dev packages
```

**Comment by ctolkien** [comment]
@Camypaper  - we've just released version 0.3.0, can you see if this issue persists?

**Comment by Camypaper** [comment]
thanks @ctolkien :)
